BuyRequestHandler.h

```diff
@@ -0,0 +1,17 @@
+/**
+ * Project MyHttpServer
+ * @author Pei-Ru Wang, Cheryl Huang, Yuan Zhou
+ * @version 0.0.1
+ */
+
+
+#ifndef _BUYREQUESTHANDLER_H
+#define _BUYREQUESTHANDLER_H
+
+#include "RequestHandler.h"
+
+
+class BuyRequestHandler: public RequestHandler {
```

Comment:
Please add class comment.
Please add member methods and variables and their respective comment.